Index: CachedSource/libkern/kxld/Makefile
===================================================================
--- CachedSource/libkern/kxld/Makefile
+++ CachedSource/libkern/kxld/Makefile
@@ -67,7 +67,7 @@ ifeq ($(strip $(SDK_DIR)),)
 endif
 
 DEFINES = -DPRIVATE
-CFLAGS=-std=c99 -Wall -Wextra -Werror -pedantic -Wformat=2 -Wcast-align \
+CFLAGS=-std=c99 -Wall -Wextra -pedantic -Wformat=2 -Wcast-align \
 	-Wwrite-strings -Wshorten-64-to-32 -Wshadow -Winit-self -Wpointer-arith \
 	-Wno-format-y2k -W -Wstrict-prototypes -Wmissing-prototypes -Wreturn-type \
 	-Wcast-qual -Wwrite-strings -Wswitch -Wcast-align -Wbad-function-cast \
Index: CachedSource/makedefs/MakeInc.def
===================================================================
--- CachedSource/makedefs/MakeInc.def
+++ CachedSource/makedefs/MakeInc.def
@@ -91,20 +91,10 @@ GENASSYM_KCC = $(CC)
 # Compiler warning flags
 #
 
-USE_WERROR := 1
-ifneq ($(BUILD_WERROR),)
-USE_WERROR := $(BUILD_WERROR)
-endif
-
-ifeq ($(USE_WERROR),1)
-WERROR := -Werror
-endif
-
 # Shared C/C++ warning flags
 WARNFLAGS_STD := \
 	-Weverything \
 	-Wextra \
-	$(WERROR) \
 	-Wpointer-arith \
 	-Wreturn-type \
 	-Wcast-qual \
Index: CachedSource/osfmk/i386/cpu_topology.c
===================================================================
--- CachedSource/osfmk/i386/cpu_topology.c
+++ CachedSource/osfmk/i386/cpu_topology.c
@@ -26,6 +26,7 @@
  * @APPLE_OSREFERENCE_LICENSE_HEADER_END@
  */

+#include <stddef.h>
 #include <mach/machine.h>
 #include <mach/processor.h>
 #include <kern/kalloc.h>
Index: CachedSource/osfmk/vm/vm_object.h
===================================================================
--- CachedSource/osfmk/vm/vm_object.h
+++ CachedSource/osfmk/vm/vm_object.h
@@ -171,7 +171,7 @@ struct vm_object {
 	int			ref_count;	/* Number of references */
 	unsigned int		resident_page_count;
 						/* number of resident pages */
-	const unsigned int	wired_page_count; /* number of wired pages
+	unsigned int	wired_page_count; /* number of wired pages
 						     use VM_OBJECT_WIRED_PAGE_UPDATE macros to update */
 	unsigned int		reusable_page_count;

