Index: CachedSource/webm2pes.cc
===================================================================
--- CachedSource/webm2pes.cc
+++ CachedSource/webm2pes.cc
@@ -8,6 +8,7 @@
 #include "webm2pes.h"
 
 #include <vector>
+#include <cstring>
 
 namespace {
 void Usage(const char* argv[]) {
@@ -258,10 +259,10 @@ bool BCMVHeader::Write(std::FILE* fileptr) const {
     std::fprintf(stderr, "Webm2Pes: BCMV write failed.\n");
   }
   const std::size_t kRemainingBytes = 6;
-  const uint8_t buffer[kRemainingBytes] = {(length >> 24) & 0xff,
-                                           (length >> 16) & 0xff,
-                                           (length >> 8) & 0xff,
-                                           length & 0xff,
+  const uint8_t buffer[kRemainingBytes] = {static_cast<uint8_t>((length >> 24) & 0xff),
+                                           static_cast<uint8_t>((length >> 16) & 0xff),
+                                           static_cast<uint8_t>((length >> 8) & 0xff),
+                                           static_cast<uint8_t>(length & 0xff),
                                            0,
                                            0 /* 2 bytes 0 padding */};
   for (std::int8_t i = 0; i < kRemainingBytes; ++i) {
